src/main/java/uk/gov/cshr/report/service/RegisteredLearnersService.java

```diff
@@ -21,6 +23,12 @@ public int deleteLearners(Collection<String> uids) {
         return registeredLearnerRepository.deleteAllByUidIn(uids);
     }
 
+    @Transactional
+    public int activateLearners(Collection<String> uids, ZonedDateTime updatedTimestamp) {
```

Comment:
Can we make this a single UID? I don't think we'll ever be sending multiple in one message

Reply:
Changes applied